packages/shared/src/contexts/AlertContext.tsx

```diff
@@ -7,12 +7,17 @@ import { apiUrl } from '../lib/config';
 
 export const ALERT_DEFAULTS: Alerts = {
   filter: false,
+  rank: true,
```

Comment:
Not sure why this one is true?

Reply:
This should be changed by the upcoming updates.